### PR TITLE
Update based on feedback

### PR DESCRIPTION
**Pull request by besated**
- Ensure all tutorials are using arctic-1
- Rename some stuff
- Add syntax highlighting to all code blocks
- style changes

### CODE REVIEW

File: pages/apis.mdx [hunk 1]
```diff
@@ -1,17 +1,19 @@
...
 
 ### Query balances gRPC
-```
+
+```proto
 // AllBalances queries the balance of all coins for a single account.
-//
 // When called from another module, this query might consume a high amount of
 // gas if the pagination field is incorrectly set.
 rpc AllBalances(QueryAllBalancesRequest) returns (QueryAllBalancesResponse) {
```
Comment (mj850) [inline pages/apis.mdx:21]: Would be extra helpful if we provided a code snippet example of how to make this gRPC call in js or go

File: pages/apis.mdx [hunk 2]
```diff
@@ -21,11 +23,13 @@ rpc AllBalances(QueryAllBalancesRequest) returns (QueryAllBalancesResponse) {
...
+
+```bash
 GET /cosmos/bank/v1beta1/balances/{address}
 ```
 
 ### Query balances RPC
-```
+
+```bash copy
 seid query bank balances {address}
```
Comment (mj850) [inline pages/apis.mdx]: Would also be extra helpful here if we included an example query with other important params such as --node so users can try this query out of the box without having to figure out seid.

**Comment by mj850** [comment]
Nit:
image
Capitalization for text after Cosmovisor, Protobuf

**Comment by mj850** [comment]
Nit:
image
Capitalization after $CONTRACT_CODE_ID:, $ACCOUNT, $LABEL, and $ADMIN_ADDRESS

Might also be extra clear to reiterate after $CONTRACT_CODE_ID that this can be found in the output of the previous command 

Also might be clear to specify that $LABEL can be used to look up the contract in future.

File: pages/apis.mdx (hunk 1 above)
Comment (besated) [reply pages/apis.mdx:21]: I don't have any code snippets to do this in go, can u create a separate pr to add this?

**Comment by besated** [comment]
Resolved all the comments from @mj850 (except the code snippet one)

**Review by philipsu522 — APPROVED** [review]
new page looks 🔥

**Pull request merged**